source/testing/testing-models.md

```diff
@@ -28,26 +28,25 @@ export default Model.extend({
 ```
 
 Now let's create a test which will call `levelUp` on the player when they are
-level 4 to assert that the `levelName` changes. We will use `moduleForModel`:
+level 4 to assert that the `levelName` changes. We will use `module` together with the `setupTest` helper method:
 
 ```tests/unit/models/player-test.js
-import { moduleForModel, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 import { run } from "@ember/runloop";
 
-moduleForModel('player', 'Unit | Model | player', {
+module('Unit | Model | player', function(hooks) {
   // Specify the other units that are required for this test.
-  needs: []
-});
-
-test('should increment level when told to', function(assert) {
-  // this.subject aliases the createRecord method on the model
-  const player = this.subject({ level: 4 });
+  test('should increment level when told to', function(assert) {
+    // this.subject aliases the createRecord method on the model
+    const player = run(() => this.owner.lookup('service:store').createRecord('player'));
 
-  // wrap asynchronous call in run loop
-  run(() => player.levelUp());
+    // wrap asynchronous call in run loop
+    run(() => player.levelUp());
```

Comment:
Is it possible to `await` this?

Reply:
I think the `run` loop here is one is good to go without `await`ing it

Reply:
Should we link to some additional info about the runloop here? We have a page at https://guides.emberjs.com/v2.18.0/applications/run-loop/ that might at least help explain it if this is one of the first times folks have heard about it ...

Reply:
Agreed. 

FWIW I’m hopefully that future work will allow us to remove all manual Ember.run wrapping (in both tests and app code) and rely on the existing systems to handle “waiting” for scheduled tasks (e.g. a rerender in xyz ms).

Reply:
I added a link to the run loop documentation in a short sentence below that code example